src/epochs.lua

```diff
@@ -598,25 +586,26 @@ end
 function epochs.distributeRewardsForEpoch(currentTimestamp)
 	local epochIndex = epochs.getEpochIndexForTimestamp(currentTimestamp - epochs.getSettings().durationMs) -- go back to previous epoch
 	local epoch = epochs.getEpoch(epochIndex)
-	if not next(epoch) then
+	if not epoch then
 		-- silently return
-		print("Not distributing rewards for last epoch. New epoch has not been created yet.")
+		print("Unable to distribute rewards for epoch. Epoch not found: " .. epochIndex)
```

Comment:
considering making this an error instead of a silent return

Reply:
Why the hesitation?

Reply:
breaking tick being the primary concern. i'd like to have this run in devnet and validate before introducing error states

Reply:
added a TODO to make this an error state - [c8472ba](https://github.com/ar-io/ar-io-network-process/pull/338/commits/c8472ba416686d262eda4e32d5819957931464d9)